extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -66,6 +66,10 @@ export class SKURecommendationPage extends MigrationWizardPage {
 		super(wizard, azdata.window.createWizardPage(constants.SKU_RECOMMENDATION_PAGE_TITLE), migrationStateModel);
 	}
 
+	private createIconTextCell(icon: IconPath, text: string): string {
```

Comment:
Same as above.

Also - you should generally avoid having functions be class members unless they need to access this. Generic functions like this would be better to put in a common module somewhere - that way multiple modules can use it and reduce code duplication.

Reply:
Noted